### PR TITLE
zfs.8 has wrong description of "zfs program -t"

### PR DESCRIPTION
**Pull request by ahrens**
### Motivation and Context

The "-t" argument to "zfs program" specifies a limit on the number of
LUA instructions that can be executed.  The zfs.8 manpage has the wrong
description.  
### Description

zfs.8 manpage should be updated to match what's in zfs-program.8

### How Has This Been Tested?

run "man zfs", inspect results.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Performance enhancement (non-breaking change which improves efficiency)
- [ ] Code cleanup (non-breaking change which makes code smaller or more readable)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation (a change to man pages or other documentation)

### Checklist:

- [x] My code follows the ZFS on Linux [code style requirements](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md#coding-conventions).
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**contributing** document](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md).
- [ ] I have added [tests](https://github.com/zfsonlinux/zfs/tree/master/tests) to cover my changes.
- [ ] All new and existing tests passed.
- [x] All commit messages are properly formatted and contain [`Signed-off-by`](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md#signed-off-by).

### CODE REVIEW

**Comment by ahrens** [comment]
@behlendorf OK, done.  I also noticed that the "zfs send" help message was not formatted correctly, so I fixed that as well.

**Comment by loli10K** [comment]
I would also suggest the following changes to detect an improperly wrapped `zfs` and `zpool` help message:

```diff
diff --git a/tests/zfs-tests/tests/functional/cli_user/misc/zfs_001_neg.ksh b/tests/zfs-tests/tests/functional/cli_user/misc/zfs_001_neg.ksh
index 1073a4030..46171caf9 100755
--- a/tests/zfs-tests/tests/functional/cli_user/misc/zfs_001_neg.ksh
+++ b/tests/zfs-tests/tests/functional/cli_user/misc/zfs_001_neg.ksh
@@ -44,16 +44,20 @@
 
 function cleanup
 {
-	if [ -e $TEST_BASE_DIR/zfs_001_neg.$$.txt ]
+	if [ -e "$TEMPFILE" ]
 	then
-		rm $TEST_BASE_DIR/zfs_001_neg.$$.txt
+		rm -f "$TEMPFILE"
 	fi
 }
 
 log_onexit cleanup
 log_assert "zfs shows a usage message when run as a user"
 
-eval "zfs > $TEST_BASE_DIR/zfs_001_neg.$$.txt 2>&1"
-log_must grep "usage: zfs command args" $TEST_BASE_DIR/zfs_001_neg.$$.txt
+TEMPFILE="$TEST_BASE_DIR/zfs_001_neg.$$.txt"
+
+eval "zfs > $TEMPFILE 2>&1"
+log_must grep "usage: zfs command args" "$TEMPFILE"
+
+log_must eval "awk '{if (length(\$0) > 80) exit 1}' < $TEMPFILE"
 
 log_pass "zfs shows a usage message when run as a user"
diff --git a/tests/zfs-tests/tests/functional/cli_user/misc/zpool_001_neg.ksh b/tests/zfs-tests/tests/functional/cli_user/misc/zpool_001_neg.ksh
index af924837a..0fddc08b2 100755
--- a/tests/zfs-tests/tests/functional/cli_user/misc/zpool_001_neg.ksh
+++ b/tests/zfs-tests/tests/functional/cli_user/misc/zpool_001_neg.ksh
@@ -45,16 +45,20 @@
 
 function cleanup
 {
-	if [ -e $TEST_BASE_DIR/zpool_001_neg.$$.txt ]
+	if [ -e "$TEMPFILE" ]
 	then
-		rm $TEST_BASE_DIR/zpool_001_neg.$$.txt
+		rm -f "$TEMPFILE"
 	fi
 }
 
+TEMPFILE="$TEST_BASE_DIR/zpool_001_neg.$$.txt"
+
 log_onexit cleanup
 log_assert "zpool shows a usage message when run as a user"
 
-eval "zpool > $TEST_BASE_DIR/zpool_001_neg.$$.txt 2>&1"
-log_must grep "usage: zpool command args" $TEST_BASE_DIR/zpool_001_neg.$$.txt
+eval "zpool > $TEMPFILE 2>&1"
+log_must grep "usage: zpool command args" "$TEMPFILE"
+
+log_must eval "awk '{if (length(\$0) > 80) exit 1}' < $TEMPFILE"
 
 log_pass "zpool shows a usage message when run as a user"
```

File: man/man8/zfs.8 [hunk 1]
```diff
@@ -4417,11 +4417,12 @@ The program can be used to gather information such as properties and
 determining if changes would succeed (zfs.check.*).
 Without this flag, all pending changes must be synced to disk before
 a channel program can complete.
-.It Fl t Ar timeout
-Execution time limit, in milliseconds.
-If a channel program executes for longer than the provided timeout, it will
-be stopped and an error will be returned.
-The default timeout is 1000 ms, and can be set to a maximum of 10000 ms.
+.It Fl t Ar instruction-limit
+Execution time limit, in number of Lua instructions to execute.
```
Comment (allanjude) [inline man/man8/zfs.8]: This still says time limit, should likely read:

Limit the number of Lua instructions to be executed.

**Comment by ahrens** [comment]
@loli10K Adding a test to verify that the help output is wrapped to 80 columns is a great idea.  I think it's out of scope for this PR though, could you open a separate one for that?

File: man/man8/zfs.8 (hunk 1 above)
Comment (ahrens) [reply man/man8/zfs.8]: Sure, fixed it here and in zfs-program.8

**Pull request merged**